feature/main/src/main/java/com/napzak/market/main/WebSocketLifecycleObserver.kt

```
@@ -1,58 +1,87 @@
+            loginStateCollectJob = activityScope.launch {
+                isLoggedIn.collectLatest { isLoggedIn ->
+                    if (isLoggedIn && isTokenAvailable()) {
+                        val storeInfo = fetchStoreInfo()
+                        val storeId = requireNotNull(storeInfo?.storeId)
+
+                        connectChatSocket(storeId)
+                        subscribeChatRooms(storeId)
                     }
                 }
             }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)